src/cockpit/packages.py

```diff
@@ -81,6 +82,11 @@ def get_libexecdir() -> str:
 
 def patch_libexecdir(obj):
     if isinstance(obj, str):
+        if '${libexecdir}/cockpit-askpass' in obj:
+            # extra-special case: we handle this internally
```

Comment:
Heh, this should probably have a `# HACK:` comment to say that this only applies to image-prepare --python`. But then that whole libexecdir thing is (hopefully) temporary, so :man_shrugging: